mobx_codegen/lib/src/store_class_visitor.dart

```diff
@@ -90,6 +92,9 @@ class StoreClassVisitor extends SimpleElementVisitor {
       ..name = element.name;
 
     _storeTemplate.observables.add(template);
+    if(_storeTemplate.generateToString) {
+      _storeTemplate.toStringList.add('${element.name}: \${${element.name}.toString()}');
```

Comment:
I would prefer keeping track of the observable field names, which we already have with the` _storeTemplate.observables`. I think we can just use that to generate the `toString()` value.

Reply:
Right that's better. I do the same with computed ?

Reply:
ya we can include computed as well